ui/stops/OBADepartureTimeLabel.h

```diff
@@ -0,0 +1,16 @@
+//
+//  OBADepartureTimeLabel.h
+//  org.onebusaway.iphone
+//
+//  Created by Chad Royal on 9/25/16.
+//  Copyright © 2016 OneBusAway. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "OBADepartureStatus.h"
```

Comment:
OBADepartureStatus.h is in OBAKit, so this import needs to change to <OBAKit/OBADepartureStatus.h>